docs/application/native/guides/notification/notification-ex.md

```diff
@@ -0,0 +1,532 @@
+
+  You can [delete notifications](#delete) when no longer needed.
+
+## Prerequisites
+
+To enable your application to use the notification-ex functionality:
+
+1. To use the Notification-ex API, the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
+    ```
+    <privileges>
+        <privilege>http://tizen.org/privilege/notification</privilege>
+        <!--To launch an application by app_control handler-->
+        <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+    </privileges>
+    ```
+
+2. To use the functions and data types of the Notification-ex API, include the `<notification_ex.h>` header file in your application:
+    ```
+    #include <notification_ex.h>
+    ```
+
+   To ensure that a notification function has been executed properly, make sure that the return is equal to `NOTI_EX_ERROR_NONE`.
+
+3. To follow the example mentioned in this guide, place an image file in the shared resource directory of your application. In addition, use the following variables in your code:
+    ```
+    char *image_path[BUFLEN];
+    char *shared_path = app_get_shared_resource_path();
+    snprintf(image_path, BUFLEN, "%stutorial_native_api_application.png", shared_path);
+    free(shared_path);
+    ```
+
+<a name="create"></a>
+## Creating notification-ex Items
+
+Each element n a notification is an item such as text, image, time and so on.
+An item can be a notification by itself, and each item can be combined to create a group notification.
+
+- Text Item:
+
+  This item can set text and hyperlinks. Text items can also be used titles and contents.
+    ```
+    ret = noti_ex_item_text_create(&text_item, "text_id", "text", "hyperlink");
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+- Image Item:
+
+  This item can set an image or an icon. Image item also contains the path information of an image.
+    ```
+    ret = noti_ex_item_image_create(&image_item, "image_id", image_path);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+- Button Item:
+
+  This item can add a button.
+    ```
+    ret = noti_ex_item_button_create(&button_item, "button_id", "button_title");
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+- Chat Message Item:
+
+  This item can create a messaging style notification. It is a combination of a text item, an image item and a time item.
+  An application can construct multiple chat message items and display them in bubble format.
+    ```
+    ret = noti_ex_item_chat_message_create(&chatmessage_item, "message_id", name_handle,
+        text_handle, image_handle, time_handle, NOTI_EX_ITEM_CHAT_MESSAGE_TYPE_USER);
+    if (ret != NOTI_EX_ERROR_NONE)
+        /* Error handling */
+    ```
+
+- CheckBox Item:
+
+  This item can provide a user-selectable checkbox and can be toggled. In addition, the select state and clear state can be determined in advance.
```

Comment:
Can we rephrase to:
This item can provide a user-selectable checkbox that can be toggled. In addition, the select state and clear state can be determined in advance.

Please verify.